packages/story-editor/src/app/rightClickMenu/useLayerSelect.js

```diff
@@ -136,15 +136,17 @@ function useLayerSelect({ menuItemProps, menuPosition, isMenuOpen }) {
     }
     return intersectingElements.map((element) => {
       const { id, isBackground, type } = element;
-      const { LayerContent } = getDefinitionForType(type);
+      const { getLayerText } = getDefinitionForType(type);
       return {
         key: id,
         supportsIcon: true,
         icon: selectedElements[0].id === id ? <Icons.CheckmarkSmall /> : null,
-        label: isBackground ? (
-          <span>{__('Background', 'web-stories')}</span>
-        ) : (
-          <LayerContent element={element} />
+        label: (
+          <span>
+            {isBackground
+              ? __('Background', 'web-stories')
+              : getLayerText(element)}
+          </span>
```

Comment:
@merapi, [it](https://github.com/GoogleForCreators/web-stories-wp/pull/11429#pullrequestreview-969268945) was actually a bug in the right-click menu, as it used the element `LayerContent` property, that I had deprecated. Good find, fixed now.